refactor_persis_layer.html

```
@@ -0,0 +1,1027 @@
+    FA["fileaudit\n~200 lines"]
+    FK["fileapikey\n~200 lines"]
+    FSS["filesession\n~650 lines"]
+    FEV["fileeventstore\n~300 lines"]
+    FM["... 19 more stores"]
+  end
+
+  subgraph FS2["Filesystem"]
+    F[("📂 File System")]
+  end
+
+  RT --> FDR
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)